.travis/publish.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+readonly tag="${TRAVIS_TAG}"
+readonly blobaccount="${DEPLOY_BLOB_ACCOUNT_NAME}"
+readonly blobkey="${DEPLOY_BLOB_ACCOUNT_KEY}"
+readonly blobcontainer="${DEPLOY_BLOB_CONTAINER}"
+readonly blobname="fortis-${tag}.jar"
+
+log() {
+  echo "$@" >&2
+}
+
+check_preconditions() {
+  if [ -z "${tag}" ]; then
+    log "Build is not a tag, skipping publish"
+    exit 0
```

Comment:
As far as I know, all that the Github releases button does is to create a tag. This is why the test tag that I created for this PR shows up as a release:

![image](https://user-images.githubusercontent.com/1086421/27645730-acfd32d0-5bdb-11e7-84eb-fd6cf30dfc9b.png)

As such, we can do either (create the release on Github or a tag on the git shell) and Travis will pick it up.